poker-pals/src/components/Tables.vue

```diff
@@ -38,39 +38,52 @@ import io from "socket.io-client";
 import UserNavbar from "./navbars/UserNavbar";
 
 export default {
-  name: 'Tables',
-  components: {
-      UserNavbar
-  },
-  created() {
+    name: 'Tables',
+    components: {
+        UserNavbar
+    },
+    props: ['authenticated', 'userID'],
+    created() {
         this.socket = io("http://localhost:3000");
-  },
-
-  data() {
-    return {
-      socket: {},
-      tables: [],
-    };
-  },
-
-  mounted() {
-    //Retrieve the current tables available and store into tables array 
-      this.socket.on("serve-tables", rooms => {
-          this.tables = rooms; 
-          console.log('These are the tables');
-          console.log(this.tables);
-
-    })
-  },
-  
-  methods: {
-    //Function to handle room joining
-    joinRoom: function (table, tableNum) {
-      //To do: Replace alert with a navigation to the poker game page (Trello)
-      alert("Joined Table " + (tableNum+1));
-      this.socket.emit("'joinRoomRequest'",table);
+    },
+    data() {
+        return {
+            socket: {},
+            rooms: [],
+        };
+    },
+    mounted() {
+        if(!this.authenticated){
+            this.$router.replace({ name: "Login" }); // send client back to login page
```

Comment:
I noticed you use `replace` for authentication failed cases only. Should we maybe change the other router navigations to also be `replace`?

Reply:
i think we should, lets address this in tomorrow's meeting